quickstarts/05-multi-agent-workflow-actors/dapr-llm-catalyst.yaml

```diff
@@ -0,0 +1,54 @@
+# https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-template/#template-properties
+version: 1
+common:
+  resourcesPath: ./components
+  logLevel: info
+  appLogDestination: console
+  daprdLogDestination: console
+
+apps:
+- appId: HobbitApp
```

Comment:
I think this needs to be `appID`

Reply:
Ah, yeah, you had mentioned this before, I'll update!